configurer/path_linux.go

```diff
@@ -24,3 +33,17 @@ func DefaultPastelExecutableDir() string {
 	homeDir, _ := os.UserHomeDir()
 	return filepath.Join(homeDir, "pastel")
 }
+
+// GetDownloadPath returns download path of the pastel executables.
+func GetDownloadPath(version string, tool constants.ToolType, architectrue constants.ArchitectureType) string {
+	var ret string
+
+	versionSubURL := constants.GetVersionSubURL(version)
+	if tool == constants.PastelD || tool == constants.RQService {
+		ret = fmt.Sprintf("%s/%s/%s-%s-%s%s", constants.DownloadBaseURL, versionSubURL, tool, "ubuntu20.04", architectrue, ".zip")
+	} else if tool == constants.SuperNode || tool == constants.WalletNode {
+		ret = fmt.Sprintf("%s/%s/%s-%s-%s%s", constants.DownloadBaseURL, versionSubURL, tool, "linux", architectrue, ".zip")
+	}
+
+	return ret
```

Comment:
if none of the conditions are met... returning err will make more sense so that every caller shouldn't check for empty return but simply catch err. 
why? because if some caller forgets to check for empty string returned from this function.. results will be unexpected and someone might waste plenty of time just trying to debug.